301-eventhub-namespace-vnet/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "eventhubNamespaceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "vnetRuleName": {
+            "value": "GEN-UNIQUE"
+        },
+        "virtualNetworkSubnetId": {
+            "value": "/subscriptions/e8baea74-64ce-459b-bee3-5aa4c47b3ae3/resourceGroups/sbehvnettest/providers/Microsoft.Network/virtualNetworks/sbehvnettest/subnets/default"
```

Comment:
use GEN value (see contribution guide)

Reply:
@bmoore-msft , I have added the parameters for VNET as per the contribution guide lines. The service accepts subnet ARM Id, have implemented the resourceId to fetch subnet Id, but throws errors as subnet name is undefined. can you please help on this?